### PR TITLE
[errors] return errors instead of exiting process

### LINKED ISSUE #74: consider using github.com/pkg/errors
this error package is nice and simple and would assist in threading error context around

### PR DESCRIPTION
**Pull request by tmc**
This opts to return error values instead of allowing sybil.Error to shut the process down via log.Fatalln.

I think the most likely way this change will introduce a regression is that it will not allow any errors in a set of records to go ignored. I would say it carries moderate risk.

Fixes #74

### CODE REVIEW

File: src/cmd/cmd_query.go [hunk 1]
```diff
@@ -231,9 +235,13 @@ func runQueryCmdLine(flags *sybil.FlagDefs) error {
 	for _, v := range allGroups {
 		switch t.GetColumnType(v) {
 		case sybil.STR_VAL:
-			loadSpec.Str(v)
+			if err := loadSpec.Str(v); err != nil {
```
Comment (okayzed) [inline src/cmd/cmd_query.go]: i don't get the two statements on one line in an if

Comment (tmc) [reply src/cmd/cmd_query.go]: it's a way to shorten LOC slightly but comes at a slight readability cost.. I can declare the err outside of the switch and deal with it afterwords.

**Comment by tmc** [comment]
circle build is broken, addressed in #81

File: src/cmd/cmd_ingest.go [hunk 2]
```diff
@@ -31,25 +30,26 @@ func ingestDictionary(r *sybil.Record, recordmap *Dictionary, prefix string) {
...
 
 		prefixName := fmt.Sprint(keyName, "_")
+		var err error
 		switch iv := v.(type) {
 		case string:
 			if INT_CAST[keyName] {
-				val, err := strconv.ParseInt(iv, 10, 64)
-				if err == nil {
-					r.AddIntField(keyName, int64(val))
+				val, cerr := strconv.ParseInt(iv, 10, 64)
```
[the remark this answers is not in the document]
Comment (tmc) [reply src/cmd/cmd_ingest.go]: if I shadow "err" then I can't populate err from line 33

**Comment by tmc** [comment]
ok changed integration target to newly cut grpc branch

**Pull request merged**